app/controllers/registrations_controller.rb

```diff
@@ -3,7 +3,8 @@ class RegistrationsController < Devise::RegistrationsController
   private
 
   def sign_up_params
-    params.require(:user).permit(:name, :email, :password, :password_confirmation)
+    password = SecureRandom.hex(10)
```

Comment:
I did not mean to have a login feature for an admin, what I meant is that I would like to use devise and the users table for both admins and users. Now users do not login, but it is likelty the they will be soon, so I wanted to keep the feature as it is with users/devise.